### PR TITLE
Using the standard connectors config API for Slack bot enabled

### PR DESCRIPTION
**Pull request by lasryaric**
PR for https://github.com/dust-tt/dust/issues/2543

I tested enabling / disabling the bot on the connector screen and on the Poke UI.
The rational for this PR is that it's a lot of useless code and it fits well within the scope of the connector config abstraction.

### CODE REVIEW

File: connectors/src/connectors/slack/index.ts [hunk 1]
```diff
@@ -629,3 +636,51 @@ export async function retrieveSlackChannelsTitles(
 
   return new Ok(titles);
 }
+
+export const getSlackConfig: ConnectorConfigGetter = async function (
```
Comment (flvndvd) [inline connectors/src/connectors/slack/index.ts:640]: Any reason to not use the function `getSlackConfig` that lives in `getSlackConfig.ts`?

**Review by flvndvd — COMMENTED** [review]
Looks great, just left one comment.

**Review by spolu — APPROVED** [review]
LGTM

File: connectors/src/connectors/slack/index.ts (hunk 1 above)
Comment (lasryaric) [reply connectors/src/connectors/slack/index.ts:640]: Forgot to remove this file. My bad.

**Pull request merged**